### ISSUE TITLE
Strange value chosen for the momentum (decay parameter) of BatchNorm2d layers

### ISSUE THREAD

**Issue by favrei** [eriklindernoren/PyTorch-YOLOv3#230]
Hi,
In models.py:43, momentum=0.9 is set.
`modules.add_module(f"batch_norm_{module_i}", nn.BatchNorm2d(filters, momentum=0.9, eps=1e-5))`

In the pytorch API reference, it warns users the naming convention of momentum could be confusing.
![image](https://user-images.githubusercontent.com/17000685/61506685-8130b980-aa15-11e9-8525-a1124ab39c23.png)
With momentum = 0.9, only 10% of the previous trained weight is kept and 90% of the statistic of the current batch is taking into account, which makes the training very unstable.

**Comment by glenn-jocher** [comment]
@killer9619 your assertion seems correct.

**Comment by peaceiris** [comment]
https://github.com/eriklindernoren/PyTorch-YOLOv3/blob/1c5a58f47c9904a9fe9903b46c73bf93a230d8e7/models.py#L43
